frontend/lib/resolution/scope-queries.cpp

```diff
@@ -764,11 +779,13 @@ bool LookupHelper::doLookupInExternBlock(const Scope* scope,
   for (auto child : ast->children()) {
     if (child->isExternBlock()) {
       bool isMethodOrField = false; // not possible in an extern block
+      bool isParenfulFunction = false; // TODO -- it could be a regular fn
```

Comment:
What does this TODO mean (how can an extern block be a regular fn), and why is it ok to hardcode this `false` for the moment?

Reply:
For now, we are scope resolving any name to return the ID of an extern block. But, we should be checking if the name is defined in the C code in the extern block. If the name is defined to be a C function, we should arguably set `isParenfulFunction=true` for it.

It's OK right now because we don't really resolve extern blocks yet in dyno (and the current workaround-y implementation to scope resolve them amounts to throwing up its hands and giving the problem to the production compiler).